modules/srvrlss-using-cluster-logging-find-logs-services-deployed.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// cluster-logging-serverless.adoc
+
+[id="using-cluster-logging-to-find-logs-for-services-deployed-with-Knative-Serving_{context}"]
+= Using Cluster Logging to find logs for services deployed with Knative Serving
+
+If Cluster Logging is deployed, you can view the OpenShift Container Platform events in Kibana.
```

Comment:
Not sure why this is referring to Openshift Container Platform events...

I'd go with something like:

"By using Openshift Cluster Logging, all logs that your applications write to the console will be collected for you in Elasticsearch. The following will show you how to apply these capabilities to applications deployed via Knative Serving specifically."